src/transformations/fix-import-directives.ts

```diff
@@ -5,41 +5,60 @@ import { Transformation } from './type';
 import { renameContract, renamePath } from '../rename';
 import { TransformerTools } from '../transform';
 
-export function* fixImportDirectives(
-  ast: SourceUnit,
-  { resolver }: TransformerTools,
-): Generator<Transformation> {
-  for (const imp of findAll('ImportDirective', ast)) {
-    const referencedSourceUnit = resolver.resolveNode('SourceUnit', imp.sourceUnit);
-
-    const aliases = imp.symbolAliases.map(a => {
-      const id = referencedSourceUnit.exportedSymbols[a.foreign.name]?.[0];
-      if (id === undefined) {
-        throw new Error(`Can't find symbol imported in ${ast.absolutePath}`);
+declare module '../transform' {
+  interface TransformData {
+    importPath: string;
```

Comment:
I'm open to a better name for this. 

Meaning is:
- for nodes that are contract definition (includes libraries and interfaces)
  - should the node not be transpiled
  - if so, where should it be imported from.
  
said otherwize:
- undefined means object should be transpiled as expected (no particular treadment)
- string means the object should not be transpiled
  - code should be removed 
  - imports in other files should like to this `importPath`
  - reference to this object should not be renamed
  
I'm also wondering if this is the right location to declare that field.

Reply:
Could be `fromPeer?: string`.

Note that it should be defined as an optional.

Reply:
- In `fix-new-statement.ts`, `isUsedInNewStatement` is not optional
- `getData` has type `(node: Node) => Partial<TransformData>;` 

So why would it be marked optional here ?